stdpopsim/catalog/HomSap_genome_data.py

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
I'm wondering if we can pull this info too? why hard code it here if we are pulling all this assembly info anyway?

Reply:
Safer this way. Plus this is being generated programatically already.

Reply:
Yeah, this file is autogenerated @andrewkern. I've added a comment header to say this, for clarity.

Reply:
sweeeet